test/CappedCrowdsale.js

```diff
@@ -62,7 +70,7 @@ contract('CappedCrowdsale', function ([_, wallet]) {
   describe('ending', function () {
 
     beforeEach(async function () {
-      await advanceToBlock(this.startBlock - 1)
+      await increaseTime(moment.duration(1, 'week'))
```

Comment:
I don't like that the time until the crowdsale starts is hardcoded here. I think we should define a function analogous to `advanceToBlock` but for time, that internally calculates how many seconds it's necessary to increase. Suppose we call it `increaseTimeTo`, then we can use it like `increaseTimeTo(this.startTime)`.